restore/restore.go

```diff
@@ -440,6 +440,12 @@ func editStatementsRedirectSchema(statements []toc.StatementWithType, redirectSc
 			replaced = true
 		}
 
+		// Statistic statements need two schema replacements
+		if strings.Contains(statement, "pg_statistic") {
+			statement = strings.ReplaceAll(statement, oldSchema, newSchema)
+			replaced = true
+		}
```

Comment:
I think, this is not cover case when old schema is "a.b", new schema is "b.a" and table has column "a.b."

Reply:
```sql
create schema "a.b";
create schema "b.a";
create schema """b.a""";
CREATE TABLE "a.b".foo(i int, j text, k bool, """a.b""." int);
INSERT INTO "a.b".foo VALUES (1, 'a', 't', 1);
INSERT INTO "a.b".foo VALUES (2, 'b', 'f', 2);
ANALYZE "a.b".foo;
```
```sh
gprestore --timestamp 20240805091034 --debug --backup-dir ~/gpbackup --on-error-continue --redirect-db gpadmin --redirect-schema '"b.a"' --include-schema '"a.b"' --with-stats
```

Reply:
it generates
```sql
...
FROM pg_attribute WHERE attrelid = '"b.a".foo'::regclass::oid AND attname = '"b.a".';
```

Reply:
I think you should use some regex like above

Reply:
hm... some values also can include 'qwe.' and you will replace it

Reply:
fixed